test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/survival/KieServerWithWorkbenchSurvivalIntegrationTest.java

```diff
@@ -148,10 +138,15 @@ private void sendRESTCallsToUnavailableKieServer() {
                                 deploymentScenario.getKieServerDeployment().getPassword()))
                 .build();
 
-        WebTarget target = httpKieServerClient.target(deploymentScenario.getKieServerDeployment().getUrl().toString() + GET_CONTAINERS_REST_ENDPOINT);
-        Response response = target.request().get();
-        response.close();
-        assertThat(response.getStatus()).isEqualTo(Response.Status.SERVICE_UNAVAILABLE.getStatusCode());
+        try {
+            URL url = new URL(deploymentScenario.getKieServerDeployment().getUrl(), KIE_CONTAINER_REQUEST_URL);
+            WebTarget target = httpKieServerClient.target(url.toString());
+            Response response = target.request().get();
+            response.close();
```

Comment:
If my memory serves me correctly, `response` is `Closable` and therefore not closing it within `finally` may potentially cause resource leaks when the request goes wrong.